item/item.go

```diff
@@ -47,9 +48,15 @@ func Unmarshal(input []byte) (Item, error) {
 	return dataUnmarshalled, nil
 }
 
+// time.time to time.Duration conversion
+func (t Item) AddHoursSincePosting() time.Duration {
+	t.HoursSincePosting = time.Duration(time.Since(time.Unix(t.UnixPostTime, 0)))
+	return t.HoursSincePosting
+}
+
 // ...
 func (t Item) RelativeTime() string {
-	elapsedHours := time.Since(time.Unix(t.UnixPostTime, 0)).Hours()
+	elapsedHours := t.HoursSincePosting.Hours()
```

Comment:
Here you can use `t.HoursSincePosting directly`, or:

```suggestion
	elapsedHours := t.HoursSincePosting
```

Reply:
I will try, just means I need to change other logic of the function as `cannot use elapsedHours (variable of int64 type time.Duration) as float64 value in argument to math.Roundcompiler[IncompatibleAssign](https://pkg.go.dev/golang.org/x/tools/internal/typesinternal#IncompatibleAssign)`

Reply:
Or I rename `HoursSincePosting` to `TimePassed` or something and call it a day and pretty much the current logic?

Reply:
No, `HoursSincePosting` is the correct name, you just need to update its type.